frontend/src/scenes/project/CreateProjectModal.tsx

```diff
@@ -37,27 +37,26 @@ export function CreateProjectModal({
 
     return (
         <LemonModal
-            title={currentOrganization ? `Create a project in ${currentOrganization.name}` : 'Create a project'}
+            title={currentOrganization ? `Create a project within ${currentOrganization.name}` : 'Create a project'}
             description={
-                <p>
-                    Most companies will want 3 projects:
-                    <br />
-                    1. Local Development
-                    <br />
-                    2. Staging
-                    <br />
-                    3. Production
-                    <br />
-                    <br />
-                    <strong>Tip:</strong> we recommend using the same project for both your website and app to track
-                    across them. You can always apply a filter to focus on just one.{' '}
-                    <Link to="https://posthog.com/manual/organizations-and-projects#projects" target="_blank">
-                        Learn more here.
-                    </Link>
-                    <br />
-                    <br />
-                    <strong>Bonus tip:</strong> you can rename your "Default Project" to "Production".
-                </p>
+                <>
+                    <p>
+                        Use Projects to organize your data into separate collections. A project usually means
+                        a completely distinct product, or an environment (production, staging, development).
+                    </p>
```

Comment:
```suggestion
```

I think we should remove this part as I think it's more confusing than just the "Tip" below. People are likely to misinterpret what a completely distinct product / environment is

Reply:
Happy with merging this after the commit above is merged

Reply:
Hmm, this removes the whole intro to the feature though. I do think it's worth explaining what a project is (with some form of "Use Projects to …"), since I've also answered my fair share of "What if I don't want to separate development data?"/"How do multiple projects work?" questions in Slack. So I'm up for any rewording suggestions – my previous point was just that it's a better experience if we suggest "prod/staging/dev" _after_ some quick of explanation. That I'd leave in somehow.
I agree "completely distinct product" might be too tricky, and can easily be harmful. I tried thinking of a good example, and I'd say something like "Facebook vs. Whatsapp" would fit – as in, two completely distinct products within one parent company. But that's huge scale and in reality those are almost separate companies, hardly applicable here. Even if someone like JetBrains signed up (with multiple different apps), it'd make much more sense to track all products and websites within one project, just to be able to analyze all the interconnections.
So I removed the "completely distinct product" mention.

Reply:
I've added some more suggestions in line with your comment - what do you think?